doc/user/content/sql/drop-index.md

```diff
@@ -6,50 +6,40 @@ menu:
     parent: 'commands'
 ---
 
-`DROP INDEX` removes an index from a materialized view. (Non-materialized views do not have any indexes.)
+`DROP INDEX` removes the specified index.
 
 ## Syntax
 
-{{< diagram "drop-index.svg" >}}
+```mzsql
+DROP INDEX [IF EXISTS] <index_name> [ 'CASCADE' | 'RESTRICT' ];
+```
 
-Field | Use
+Options | Description
```

Comment:
Could you do a pass to consistently change this for all reference pages? Just so we don't end up with half the pages using `Field` and `Use`, and the other half saying `Option` and `Description`.

Reply:
Sure!

Reply:
Oh ... to clarify, that pass will be in a separate PR. Am noticing that we use
"Field" for both user-input fields and options. Sometimes "Name" and sometimes "Options" 😄 

Also, noticed that sometimes, we specify `Disregarded.` -- We maybe want to adopt that (or Ignored or some such) for things like `Cascade` and `Restrict` in `Drop index`.

Reply:
Sounds good! Not surprised that there is at least one other variation. 🫠 Always good to do a sweep every now and then!